src/smpp_gateway/client.py

```diff
@@ -97,17 +109,26 @@ def _save_delivery_receipt(self, pdu, params):
             mtmessagestatus__message_id=params["receipted_message_id"],
         ).update(
             modify_time=timezone.now(),
-            status="delivered",
+            status=MTMessage.Status.DELIVERED,
```

Comment:
Using [these enumeration types](https://docs.djangoproject.com/en/3.2/ref/models/fields/#enumeration-types) for status fields across the library.